### ISSUE TITLE
Episode 7 - Remotes in GitHub: Update object/bytes references

### ISSUE THREAD

**Issue by kekoziar** [swcarpentry/git-novice#625]
Regarding
~~~
Counting objects: 9, done.
Delta compression using up to 4 threads.
Compressing objects: 100% (6/6), done.
Writing objects: 100% (9/9), 821 bytes, done.
Total 9 (delta 2), reused 0 (delta 0)
To https://github.com/vlad/planets
 * [new branch]      master -> master
Branch master set up to track remote branch master from origin.
~~~
Because of addition to the lesson (specifically, adding spaceships) objects have changed such that the counts are now

~~~
Enumerating objects: 16, done.
Counting objects: 100% (16/16), done.
Delta compression using up to 8 threads
Compressing objects: 100% (11/11), done.
Writing objects: 100% (16/16), 1.45 KiB | 372.00 KiB/s, done.
Total 16 (delta 2), reused 0 (delta 0)
remote: Resolving deltas: 100% (2/2), done.
To https://github.com/vlad/planets.git
 * [new branch]      master -> master
 ~~~

While the output will be different based on what exercises have been completed, IMO we should have the main text of the lesson be as true as possible to the output of the lesson sans exercises.

Also, my output does not have the last line, 
 "Branch master set up to track remote branch master from origin"
 can someone verify if this is a change to git or something else?

**Comment by munkm** [comment]
I did a little dummy example and found that I got a variant of the last line if I executed 
```
$ git push -u origin master
Counting objects: 6, done.
Delta compression using up to 4 threads.
Compressing objects: 100% (3/3), done.
Writing objects: 100% (6/6), 420 bytes | 420.00 KiB/s, done.
Total 6 (delta 0), reused 0 (delta 0)
To github.com:munkm/testrepo.git
 * [new branch]      master -> master
Branch master set up to track remote branch master from origin by rebasing.
```

Whereas when I didn't use the upstream option I didn't get that message

```
$ git push origin master
Counting objects: 6, done.
Delta compression using up to 4 threads.
Compressing objects: 100% (3/3), done.
Writing objects: 100% (6/6), 420 bytes | 420.00 KiB/s, done.
Total 6 (delta 0), reused 0 (delta 0)
To github.com:munkm/testrepo.git
 * [new branch]      master -> master 
```

**Comment by kekoziar** [comment]
Hmmm.  Upstream isn't introduced until later in that episode in the call-out, **The ‘-u’ Flag**.

**Comment by munkm** [comment]
That said, we could put a note in the instructor guide that if users copy the github instructions of syncing an existing repo they will probably get this line, since it includes the -u flag.

**Comment by munkm** [comment]
Closed via swcarpentry/git-novice#655
